### DISCUSSION TITLE
Possibility to change where Image Transform folders are stored?

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#8136] [Ideas]
> Created by: Mark Bowen (markbowen@helitorque.com) on 2015/05/14 10:44:03 +0000
> Votes at time of UserVoice import: 14

-----------------------------------

Would be nice to have a configuration option for the Image Transformation folders to be placed somewhere other than in the same folder as the original images.

Currently if you have say :

/images/image.jpg

and you apply a transformation to the image then you will end up with something like :

/images/_300x300_crop_top-center_75/image.jpg

With the way this is currently if you want to upload large images and not have anyone able to access them then this isn't possible as they can just delete the _300x300_crop_top-center_75 part from the URL and they get the original full sized images.

Would be nice to place the transformation folders somewhere other than the same folder so that people don't know where they originated from.

Best wishes,

Mark

**Comment by angrybrad** [comment]
> Posted by Stephen McIver (spm999@gmail.com) on 2015/04/13 23:58:09 +0000
-----------------------------------

Hoping that P&amp;T can add this as a config option. I&#39;m not keen on the transforms being stored in the same place as the original assets as it means that I can&#39;t quickly see the size of my original assets folder easily, and when copying assets folders, I&#39;m ending up with folders full of transformed images unless I manually pick them out, which can be tedious.

So I&#39;d much rather have my original assets in one folder location, and have my transforms in another folder location. Much neater and tidier that way, in my opinion.

**Comment by wbrowar** [comment]
Hey Brad and team, I think this is a good FR to revisit now with the CMS going further into the headless direction. I'm finding the need to do things like copy the transformed images of certain volumes out of the uploads folder to move on to a front-end server so I don't need to move un-transformed source images from the back-end to the front-end.

**Comment by brandonkelly** [comment]
@wbrowar Yep, we’re going to need to sort this out for Craft Cloud :)
